src/tasks/processIssueComment/probot-processIssueComment.js

```diff
@@ -141,14 +152,18 @@ async function probotProcessIssueComment({ context, commentReply, analytics }) {
             branchName,
         })
         const optionsConfig = await setupOptionsConfig({ repository })
+        const oldContributions = findOldContributions(optionsConfig, safeWho)
```

Comment:
How do you feel about moving all this logic, into 
https://github.com/all-contributors/all-contributors-bot/blob/master/src/tasks/processIssueComment/OptionsConfig/index.js#L74

Would make it easier to unit test too